app/views/course/material/folders/show.html.slim

```diff
@@ -1,4 +1,4 @@
-= page_header(@folder.name)
+= page_header(display_name(@folder))
```

Comment:
but very likely you may need to have a global helper, because you want to link to it.